### PR TITLE
tasks.py: fix flake8-reported issues

### PR DESCRIPTION
**Pull request by fcami**
Fixes: https://pagure.io/freeipa/issue/8931
Signed-off-by: François Cami <fcami@redhat.com>

### CODE REVIEW

File: ipatests/pytest_ipa/integration/tasks.py [hunk 1]
```diff
@@ -2122,8 +2124,8 @@ def create_active_user(host, login, password, first='test', last='user',
         result = host.run_command(
             "KRB5_TRACE=/dev/stdout kinit %s" % login,
             stdin_text='{0}\n{1}\n{1}\n'.format(
-                temp_password, password, raiseonerr=False
-            )
+                temp_password, password
+            ), raiseonerr=False
         )
```
Comment (miskopo) [inline ipatests/pytest_ipa/integration/tasks.py:2129]: Funny it wasn't caught earlier

**Comment by miskopo** [comment]
These are good changes. I really like them. ACK

File: ipatests/pytest_ipa/integration/tasks.py (hunk 1 above)
Comment (fcami) [reply ipatests/pytest_ipa/integration/tasks.py:2129]: Yes, that's what I was thinking too.

**Comment by fcami** [comment]
Thanks for the review @miskopo

**Comment by fcami** [comment]
master:

* 4618c3c24ab2405a89b7289321ca021765ba733d tasks.py: fix flake8-reported issues

**Pull request closed without merging**